media-libs/libgroove/libgroove-4.3.0.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit cmake-utils
+
+DESCRIPTION="Streaming audio processing library."
+HOMEPAGE="https://github.com/andrewrk/libgroove"
+SRC_URI="https://github.com/andrewrk/libgroove/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
```

Comment:
We should probably provide a SUBSLOT, in this case `SLOT="0/4"` also see https://wiki.gentoo.org/wiki/Sub-slots_and_Slot-Operators